packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -630,6 +635,18 @@ class ThemeData extends Diagnosticable {
   /// [primaryTextTheme], and [accentTextTheme].
   final Typography typography;
 
+  /// Components of the [CupertinoThemeData] to override from the Material
```

Comment:
If there were a CupertinoThemeData.merge() method, roughly like [TextStyle's merge](https://docs.flutter.io/flutter/painting/TextStyle/merge.html) then you could just say that the default  MaterialBasedCupertinoThemeData will be merged with the cupertinoOverrideTheme.

Reply:
I'm worried it might create more confusion on having to explain the difference between a CupertinoThemeData.merge, a CupertinoThemeData.copyWith, a MaterialBasedCupertinoThemeData.merge and a MaterialBasedCupertinoThemeData.copyWith.